src/manifest/msquic.clog.cs

```diff
@@ -37,44 +38,48 @@ public struct SocketAddress
             public ulong S_v6Addr2;
         };
 
-        public static string QUIC_FLAGS(uint value)
-        {
-            return "QUIC_FLAGS (good!) " + value;
-        }
-
-        public static string QUIC_IP_ADDR(byte[] value)
+        public static string ADDR(byte[] value)
         {
             int len = value.Length;
-            IntPtr i = System.Runtime.InteropServices.Marshal.AllocHGlobal(len);
-            System.Runtime.InteropServices.Marshal.Copy(value, 0, i, len);
+            IntPtr i = Marshal.AllocHGlobal(len);
             string msg = "";
-            SocketAddress sa2 = (SocketAddress)System.Runtime.InteropServices.Marshal.PtrToStructure(i, typeof(SocketAddress));
+            try 
+            {
+                Marshal.Copy(value, 0, i, len);
+                SocketAddress sa2 = (SocketAddress)Marshal.PtrToStructure(i, typeof(SocketAddress));
+            }
+            finally
+            {
+                Marshal.FreeHGlobal(i);
+            }
 
             switch (sa2.si_family)
             {
                 case 0:  //<--unspecified
                     msg += "Unspecified";
                     break;
-                case 23: //<--v6
-                    msg += "IPV6: sin6_flowinfo=" + sa2.sin6_flowinfo + " port=" + sa2.sin_port + "part1=" + sa2.S_v6Addr1 + ", part2=" + sa2.S_v6Addr2;
+                case 2:  //< --v4
+                    msg += $"IPV4:{sa2.S_addr}:{sa2.sin_port}";
```

Comment:
We don't want the "IPV4:" or `IPV6:" text here. Just the normal text, like `127.0.01:443` or `[::1]:443`. Also, for unspecified, I'd like to log `:<port>` or `*:<port>` to indicate a wildcard.